CHANGELOG.md

```diff
@@ -7,6 +7,107 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
+-   Updates supabase example app to use supertokens-auth-react version 0.24
+-   Updates grammar for "Something went wrong" error text.
+
+### Added
+
+-   Added `overrideGlobalClaimValidators` callback to `SessionAuth`
+-   Added `validateClaims` and `getInvalidClaimsFromResponse` to the `Session` recipe
+-   Added `API_INVALID_CLAIM` event to the `Session` recipe
+-   Export for `EmailVerification` recipe
+
+### Fixes
+
+-   Fixed typing of `onSuccess` prop on passwordless `SignInUpProps`
+-   Fixed some full-page redirect that happened even when using react-router-dom
+
+### Breaking changes
+
+-   EmailVerification recipe is now not initialized as part of auth recipes. You can add it to the recipe list as `EmailVerification.init` like other recipes.
+-   Removed `OFF` from possible `EmailVerification` recipe modes, default is updated to `REQUIRED`.
+-   Moved email verification related events, overrides, pre-api hooks and redirection contexts into the `EmailVerification` recipe. You should configure them while initializing the `EmailVerification` recipe.
+-   Removed recipe specific auth wrapper components. You should use `SessionAuth` instead.
+-   Removed email verification methods from auth recipes. You should now call them on the `EmailVerification` recipe directly.
+-   `requireAuth` now defaults to true in `SessionAuth` to match the behavior of the removed recipe specific wrappers.
+-   Removed `redirectToLogin` from `SessionAuth`.
+-   Moved `redirectToAuth` to SuperTokens out of auth recipes. You should now call `SuperTokens.redirectToAuthWithoutRedirectToPath()` for the same behavior.
```

Comment:
This will change to calling `SuperTokens.redirectToAuth` (see my comment in relevant PR)